docs/suspensive.org/src/pages/en/docs/jotai/AtomValue.mdx

```diff
@@ -10,25 +10,21 @@ You can use Jotai's atom as is.
 
 ```tsx /AtomValue/
 import { AtomValue } from '@suspensive/jotai'
-import { atom } from "jotai";
+import { atom } from 'jotai'
 
-const countAtom = atom(1);
+const countAtom = atom(1)
 
-const Example = () => (
-  <AtomValue atom={countAtom}>
-    {(count) => (
-      <>count: {count}</>
-    )}
-  </AtomValue>
-)
+const Example = () => {
+  return <AtomValue atom={countAtom}>{(count) => <>count: {count}</>}</AtomValue>
+}
```

Comment:
The Prettier rules have been applied, resulting in the following changes. Is this acceptable? To avoid these changes, you can add `<!-- prettier-ignore -->`. However, this will prevent linting from being applied to that file.

[BEFORE]

```ts
const Example = () => (
  <AtomValue atom={countAtom}>
    {(count) => (
      <>count: {count}</>
    )}
  </AtomValue>
)
```

[AFTER]
```ts
const Example = () => {
  return <AtomValue atom={countAtom}>{(count) => <>count: {count}</>}</AtomValue>
}
```

Reply:
In my personal opinion, I would prefer that the examples not have Prettier applied. The goal is to present the <AtomValue/> in a way that enhances readability for users. How about leaving the examples unformatted to present them in a way that users can easily understand?

Reply:
In my personal opinion, how about applying a new "prettier" format focused on readability for the /docs/suspensive.org/** path? Introducing this formatter would allow us to establish clear guidelines, creating a better environment where both new contributors and existing participants can collaborate effectively.

Reply:
But I want to get benefit of mdx auto-format

So, how about updating prettier's config to reduce length of each code line

Reply:
> In my personal opinion, how about applying a new "prettier" format focused on readability for the /docs/suspensive.org/** path? Introducing this formatter would allow us to establish clear guidelines, creating a better environment where both new contributors and existing participants can collaborate effectively.

I think it's proper way without big change

How do you guys think of it? @kangju2000 @gwansikk

Reply:
> So, how about updating prettier's config to reduce length of each code line
> In my personal opinion, how about applying a new "prettier" format focused on readability for the /docs/suspensive.org/** path?

I agree. That’s a great idea. It would be good to apply the appropriate Prettier configuration to the parts that have examples through the path.

Reply:
Okay. I'll try adding some custom Prettier rules for our documents to improve readability while keeping the existing format mostly intact.